src/config.c

```diff
@@ -618,6 +618,26 @@ int parseCommandLine(int argc,char **argv,ParodusCfg * cfg)
     return 0;
 }
 
+void free_cfg(ParodusCfg *cfg)
+{
+    if (cfg->mtls_client_cert_path != NULL )
+         free(cfg->mtls_client_cert_path);
```

Comment:
Set these pointers to null after free, as pointed by the "Use After free" vulnerability

Reply:
yes try to fix security warnings reported .
Please submit PR to master branch as that is the latest branch which is pulled in yocto builds.